libs/feature/search/src/lib/utils/mapper/elasticsearch.mapper.spec.ts

```diff
@@ -237,80 +237,92 @@ describe('ElasticsearchMapper', () => {
           id: '11700',
           links: [
             {
-              description: '',
+              label: 'La base de données Quadrige',
               name: 'La base de données Quadrige',
               protocol: 'WWW:LINK',
               url: 'https://wwz.ifremer.fr/envlit/Quadrige-la-base-de-donnees',
             },
             {
-              description: '',
+              label: 'La surveillance du milieu marin et côtier',
               name: 'La surveillance du milieu marin et côtier',
               protocol: 'WWW:LINK-1.0-http--link',
               url: 'https://wwz.ifremer.fr/envlit/Surveillance-du-littoral',
             },
             {
               description:
                 'Manuel pour l’utilisation des données REPHY. Informations destinées à améliorer la compréhension des fichiers de données REPHY mis à disposition des scientifiques et du public. ODE/VIGIES/17-15. Ifremer, ODE/VIGIES, Coordination REPHY & Cellule Quadrige (2017).',
+              label:
+                'Manuel pour l’utilisation des données REPHY. Informations destinées à améliorer la compréhension des fichiers de données REPHY mis à disposition des scientifiques et du public. ODE/VIGIES/17-15. Ifremer, ODE/VIGIES, Coordination REPHY & Cellule Quadrige (2017).',
               name: 'Manuel pour l’utilisation des données REPHY',
               protocol: 'WWW:LINK',
               url: 'http://archimer.ifremer.fr/doc/00409/52016/',
             },
             {
               description: 'Lieu de surveillance (point)',
+              label: 'Lieu de surveillance (point)',
               name: 'surval_parametre_point',
               protocol: 'OGC:WMS',
               url: 'http://www.ifremer.fr/services/wms/surveillance_littorale',
             },
             {
               description: 'Lieu de surveillance (point)',
+              label: 'Lieu de surveillance (point)',
               name: 'surval_parametre_point',
               protocol: 'OGC:WFS',
               url: 'http://www.ifremer.fr/services/wfs/surveillance_littorale',
             },
             {
               description: "Extraction des données d'observation",
+              label: "Extraction des données d'observation",
```

Comment:
I tried running `npm run format`, but strangely it does not seem to get rid of double quotes here anymore?

Reply:
ah, of course I didn't notice the apostrophe